src/objective-c/GRPCClient/private/GRPCReachabilityFlagNames.xmacro.h

```diff
@@ -65,3 +65,4 @@ GRPC_XMACRO_ITEM(interventionRequired, InterventionRequired)
 GRPC_XMACRO_ITEM(connectionOnDemand, ConnectionOnDemand)
 GRPC_XMACRO_ITEM(isLocalAddress, IsLocalAddress)
 GRPC_XMACRO_ITEM(isDirect, IsDirect)
+GRPC_XMACRO_ITEM(isWWAN, IsWWAN)
```

Comment:
I'd suggest renaming that one to `isWWAN`, so next time someone goes looking for it it's easier to spot.